### PR TITLE
Alerts resolution support

### PR DESCRIPTION
**Pull request by arikalon1**
Supporting alert resolution on the Robusta UI
Added 3 new columns: starts_at, updated_at, ends_at - all nullable
Added the alert fingerprint to the finding (this column already exists)
Try to validate the change is indeed backward compatible

### CODE REVIEW

File: src/robusta/core/reporting/base.py [hunk 1]
```diff
@@ -94,6 +95,9 @@ def __init__(
         finding_type: FindingType = FindingType.ISSUE,
         failure: bool = True,
         creation_date: str = None,
+        fingerprint: str = None,
+        starts_at: datetime = datetime.now(),
```
Comment (aantn) [inline src/robusta/core/reporting/base.py:99]: There is a bug here. Default parameters in Python are evaluated only once, so `starts_at` will always be equal to the time when this function was first imported, not the time of the function call.

You can try this for yourself (see below):
Screen Shot 2022-06-09 at 20 14 30

Comment (aantn) [reply src/robusta/core/reporting/base.py:99]: The common solution is to set `start_at=None` and in the function itself if it is None then you set to `datetime.now()`

File: src/robusta/utils/parsing.py [hunk 2]
```diff
@@ -13,3 +14,10 @@ def load_json(s: Union[str, bytes]) -> Any:
         sep = "----------------"
         logging.error(f"could not parse json:\n{sep}\n{s}\n{sep}\n")
         raise
+
+
+def to_datetime_str(datetime_obj: datetime.datetime) -> str:
```
Comment (aantn) [inline src/robusta/utils/parsing.py:19]: I would give this a slightly more descriptive name like `datetime_to_db_str`. The comment is very good, but I would make the name a little better.

**Review by aantn — APPROVED** [review]
Found a bug (I think). Other than that looks good.

File: src/robusta/core/reporting/base.py (hunk 1 above)
Comment (arikalon1) [reply src/robusta/core/reporting/base.py:99]: oops. Good catch!
Fixing

File: src/robusta/utils/parsing.py (hunk 2 above)
Comment (arikalon1) [reply src/robusta/utils/parsing.py:19]: fixing

**Pull request merged**